aikau/src/main/resources/alfresco/services/DialogService.js

```diff
@@ -474,8 +474,8 @@ define(["dojo/_base/declare",
             duration: payload.duration || 0,
             fullScreenMode: payload.fullScreenMode || false,
             fullScreenPadding: !isNaN(payload.fullScreenPadding) ? payload.fullScreenPadding : 10,
-            widgetsContent: payload.widgetsContent,
-            widgetsButtons: payload.widgetsButtons,
+            widgetsContent: lang.clone(payload.widgetsContent),
+            widgetsButtons: lang.clone(payload.widgetsButtons),
```

Comment:
Just to double-check that the widgetsContent and widgetsButtons are either never undefined/null or do not cause a problem with lang.clone if they are?